lang/en/texts/index-pro.html

```diff
@@ -0,0 +1,39 @@
+<h1>Open Food Facts for producers</h1>
+
+<div class="row hide-when-logged-in">
+
+<div class="medium-12 columns">
+
+<p>A completely free platform to allow manufacturers to easily manage the data and photos of their products on Open Food Facts.</p>
+
+<h2>Make your products visible in more than 100 apps and services</h2>/
```

Comment:
mention scientific research ?

Reply:
It's not really an incentive for producers unfortunately.